client/post-editor/editor-drawer/index.jsx

```diff
@@ -139,7 +139,10 @@ const EditorDrawer = React.createClass( {
 
 	renderSharing: function() {
 		return (
-			<EditorSharingAccordion
+			<CodeSplitRender
+				require={ function( callback ) {
+					require( [ 'post-editor/editor-sharing/accordion' ], callback );
```

Comment:
>Usually webpack is using require.ensure to inject split points

It's not entirely obvious, but this is the AMD (not CommonJS) syntax for asynchronously loading modules (note the callback). [Both are supported by Webpack](https://webpack.github.io/docs/code-splitting.html#defining-a-split-point) in defining a split point. The reason for using AMD is that it provides the loaded modules as arguments to the callback function, whereas CommonJS passes only `require`, and the `CodeSplitRender` function is otherwise not capable of determining which modules it should proceed to load (unless we duplicated the modules array as a separate prop).